awsconfigs/common/istio-ingress/base/ingress.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   - http:
       paths:
       - backend:
-          serviceName: istio-ingressgateway
-          servicePort: 80
+          service:
+            name: istio-ingressgateway
+            port:
+              number: 80
         path: /*
+        pathType: ImplementationSpecific
```

Comment:
How did you pick the value for this field ?

Reply:
From kubernetes documentation, `pathType` is now required for each specified path. Options are `Prefix`, `Exact`, and `ImplementationSpecific`. To match the undefined `v1beta1` behavior, use `ImplementationSpecific`. I went with the choice to match the behavior from v1beta1 which we were using previously.

Reply:
doc link? just to give read it

Reply:
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122